line-bot-api-client/src/main/java/com/linecorp/bot/client/LineOAuthClient.java

```diff
@@ -35,6 +37,32 @@ static LineOAuthClientBuilder builder() {
         return new LineOAuthClientBuilder();
     }
 
+    /**
+     * Issues a channel access token. This method lets you use JWT assertion for authentication.
+     *
+     * <p>You can issue up to 30 tokens.
+     * If you reach the maximum limit, additional requests of issuing channel access tokens are blocked.
+     *
+     * @param clientAssertion A JSON Web Token the client needs to create and sign with the private key created
+     *                        when issuing an assertion signing key.
+     * @see <a href="https://developers.line.biz/en/reference/messaging-api/#issue-channel-access-token-v2-1">Issue channel access token v2.1</a>
+     */
+    CompletableFuture<IssueChannelAccessTokenResponse> issueChannelTokenByJWT(
+            @Field("client_assertion") String clientAssertion);
```

Comment:
We don't need to add `@Field` annotation here, isn't it?

Reply:
🙏 done.